operations/mimir/config.libsonnet

```diff
@@ -100,20 +100,29 @@
 
     store_gateway_replication_factor: 3,
 
-    memcached_frontend_enabled: true,
-    memcached_frontend_max_item_size_mb: 5,
+
+    cache_frontend_enabled: true,
+    cache_frontend_max_item_size_mb: 5,
+    cache_frontend_backend: 'memcached',
+    cache_frontend_name: 'memcached-frontend',
     memcached_frontend_mtls_enabled: false,
 
-    memcached_index_queries_enabled: true,
```

Comment:
WDYT about preserving the old config for a while to make the migration (for us) easier?